src/app/view/_routes/editorganization/editorganization.component.ts

```diff
@@ -37,8 +49,10 @@ export class EditOrganizationComponent implements OnInit {
   constructor(
     private readonly formBuilder: FormBuilder,
     private readonly organizationService: OrganizationService,
+    private readonly administratorService: AdministratorService,
```

Comment:
mi pare che questo componente inizi ad avere un po' troppe dipendenze, non ha più senso fare un componente separato per admin management?

Reply:
ora è finito, non dovrebbero più aggiungersi altre dipendenze o codice extra in teoria

Reply:
a parte eventualmente la get degli administrator appena avremo l'endpoint

Reply:
ok questo lo segno risolto allora